pages/pledge/[id].page.js

```diff
@@ -94,9 +101,13 @@ const PledgePage = () => {
             <Text>Revenue: ${campaign.fiatAmount}</Text>
             <Text>Pledged: ${campaign.currentAmount[0].amount}</Text>
 
-            <Button onClick={() => setIsModalOpen((prev) => !prev)}>
-              Pledge Now
-            </Button>
+            {isOwner ? (
```

Comment:
I think we need another condition that is: the goal is reached and the time ended 

but maybe you want to include that on another pr

Reply:
Yes, I was thinking of adding it to another PR. This one does not take into account the UX of the claim, only the connection with the contract.